src/sportsradar/transform/draftfeeds.py

```diff
@@ -0,0 +1,48 @@
+from src.sportsradar import logging_helpers
+
+logger = logging_helpers.get_logger(__name__)
+
+
+class DraftFeedsTransformer:
+    """
+    DraftFeedsTransformer class is used to transform draft feeds data.
+
+    Args:
+        data (dict): A dictionary containing draft feeds data.
+
+    Methods:
+        transform_weekly_depth_charts(): Transforms the weekly depth charts data by removing unwanted feeds.
```

Comment:
please update it to relevant documentation

Reply:
Documentation Updated @ggurjar333